packages/teams-js/src/public/profile.ts

```diff
@@ -164,6 +161,11 @@ export type Rectangle = {
   height: number;
 };
 
+/**
+ * @hidden
+ * An internal representation of the showProfile parameters suitable for sending via postMessage.
+ * The hub expects to receive an object of this type.
+ */
```

Comment:
Does this need a beta tag? (Likely outside the scope of your PR, but just noticed it)

Reply:
Since it's hidden I don't think it' s a big deal (no dev docs will get generated for this) but I will add it for correctness!